helix-core/src/test/java/org/apache/helix/controller/rebalancer/waged/model/AbstractTestClusterModel.java

```diff
@@ -38,7 +37,7 @@
 import org.mockito.Mockito;
 import org.testng.annotations.BeforeClass;
 
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
Still wildcard. I think @pkuwm has already added it to style file. Please try importing the latest file.

Reply:
Fixed: `import static org.mockito.Mockito.when;`